EpiResponsivePicture/ResizedImage/Property/JsonFocalPointConverter.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using EPiServer.ServiceLocation;
+using Newtonsoft.Json;
+
+namespace Forte.EpiResponsivePicture.ResizedImage.Property
+{
+    [ServiceConfiguration(typeof(JsonConverter))]
```

Comment:
Are you sure it won't brake anything? If I understand it correctly it registers globally `JsonFocalPointConverter` as implementation of `JsonConverter`. What if it will be injected in other class which requires `JsonConverter` and this class will attempt to read json and try to treat it as `FocalPoint` ?

Reply:
IMO it's kind of common practice, it just registers custom JSON serializer for Episerver editing purposes:
https://world.episerver.com/forum/developer-forum/-Episerver-75-CMS/Thread-Container/2014/2/Custom-Property-EditorValue-Serialization/

Reply:
👍